### PR TITLE
Update build/installation instructions

### PR DESCRIPTION
**Pull request by itkach**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+build-linux64-static-tar: build-linux64-static
+	tar -C ${DIST} -czvf ${LINUX64_STATIC_TAR} ${LINUX64_OUTPUT_DIR}
+
+build-docker-image: build-linux64-static
+	docker build . --tag ${NAME} --tag ${DOCKER_TAG}
+
+build-docker-image-tar: build-docker-image
+	docker save ${DOCKER_TAG} | gzip --best -c > ${DOCKER_IMAGE_TAR}
+
+build-local:
```
Comment (michaeljklein) [inline Makefile]: How about a `build-osx` that checks that you're on OSX before running `build-local`?

File: Makefile [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+	mkdir -p ${FULL_OUTPUT_DIR}
+	stack install --local-bin-path ${FULL_OUTPUT_DIR}
+
+build-local-tar: build-local
+	tar -C ${DIST} -czvf ${LOCAL_TAR} ${OUTPUT_DIR}
+
+build-all: build-linux64-static-tar build-docker-image-tar build-local-tar
+
+release-candidate: build-all
```
Comment (michaeljklein) [inline Makefile]: Most releases I've seen on GitHub also include hashes of each binary (as separate files), could we include those?

File: README.md [hunk 3]
```diff
@@ -8,44 +8,113 @@ It also contains CLI interfaces for working with Lorentz contracts:
...
-[`stack v1.9.3`](https://github.com/commercialhaskell/stack/releases/tag/v1.9.3).
+### Use Pre-Built Binaries
+
+*Note:* Check https://github.com/tqtezos/lorentz-contract-param/releases/ for
+latest release versions
+
+### Linux (64-bit)
+
+``` bash
+wget https://github.com/tqtezos/lorentz-contract-param/releases/download/v1.2.2.1.0-1/lorentz-contract-param-1.2.2.1.0-1-linux-x86_64-static.tar.gz
```
Comment (michaeljklein) [inline README.md]: Can the `1.2.2.1.0-1` string be a bash variable so this readme doesn't need to be updated for new tags?

File: Makefile (hunk 2 above)
Comment (itkach) [reply Makefile]: Done

File: Makefile (hunk 1 above)
Comment (itkach) [reply Makefile]: I had the same thought, but then it wasn't immediately clear to me how to implement in the Makefile so I thought about it some more and decided it doesn't seem to be solving any actual problem :) I'd rather skip it for now

**Pull request merged**